CONTRIBUTING.md

```diff
@@ -21,51 +21,6 @@ or that there are particular issues that you should know about before implementi
 You will have to fork the `apm-server` repo,
 please follow the instructions in the [readme](README.md).
 
-### Submitting your changes
-
-Generally,
-we require that you test any code you are adding or modifying.
-Once your changes are ready to submit for review:
-
-1. Sign the Contributor License Agreement
-
-    Please make sure you have signed our [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
-    We are not asking you to assign copyright to us,
-    but to give us the right to distribute your code without restriction.
-    We ask this of all contributors in order to assure our users of the origin and continuing existence of the code.
-    You only need to sign the CLA once.
```

Comment:
I think this is important information. Can you add it as a comment to the PR template?

Reply:
Oh I see. I'll add it alongside the CLA check.